src/main/java/org/schemaspy/model/Catalog.java

```diff
@@ -61,5 +58,16 @@ public int compareTo(Catalog i) {
     public String toString() {
         return name;
     }
-    
+
+	@Override public boolean equals(Object o) {
+		if (this == o) return true;
+		if (o == null || getClass() != o.getClass()) return false;
+		Catalog catalog = (Catalog) o;
+		return Objects.equals(name, catalog.name);
+	}
+
+	@Override public int hashCode() {
+
+		return Objects.hash(name);
```

Comment:
IntelliJ complains on this line about "Object instantiation inside `hashCode()` (varargs call)".
Should not make that much of a difference but I propose that a `name.hashCode()` is enough here. ;-)